src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,74 @@
+package lotto.controller;
+
+import lotto.domain.LastWeekWinnerNumber;
+import lotto.domain.LottoGames;
+import lotto.domain.RankingResult;
+import lotto.domain.RateOfReturn;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+public class LottoController {
+
+	private final static int PRICE_PER_GAME = 1000;
+	private RankingResult rankingResult;
+	private double rateOfReturn;
+
+	public LottoController() {
+	}
+
+	public void run() {
+		try {
+			int purchasePrice = getPrice();
+			validationLottoGamesSize(purchasePrice / PRICE_PER_GAME);
+			LottoGames lottoGames = getLottoGames(purchasePrice / PRICE_PER_GAME);
```

Comment:
PRICE_PER_GAME 이란 값과 유효한 게임 사이즈인지 확인하는 로직은
LottoGames가 관리하고 수행해야할 책임 같아요 :)

```suggestion
			LottoGames lottoGames = getLottoGames(purchasePrice);
```

Reply:
처음에  LoottoGames에 해당 역할을 부여야하려고 했으나,

 `	public LottoGames(int gameSize) {
		this(IntStream.rangeClosed(1, gameSize) `
this 호출전에는 체크를 못하는데, gameSize가 정상적이지 못할경우 rangeClosed에서 문제가 발생해서 고민하다가 Controller로 옮겼는데요.
this를 호출해서 다르

Reply:
this 호출전에 코드를 실행할 수 없기에 컨트롤러로 할지 고민을 하셨군요 👍 👍 
말씀해주신대로 this로 다른 생성자를 호출하지 않고 별도로 로직을 구현해주시면 좋을 것 같습니다.
정적 팩터리 메서드를 이용해도 좋을 것 같아요 :)